test/test.js

```diff
@@ -601,8 +609,7 @@ suite('Methods', function() {
     })
   })
 
-  // TODO: Waiting to verify behavior
```

Comment:
Is this change OK? I'm not sure what this TODO was for, but as far as I can tell the tests appear correct and inline with the spec and current implementations. I needed to enable them to verify that the Request constructor throws under the same conditions I'm using to verify that fetch() rejects.